src/unwinder/arm64/unwinder_arm64.cpp

```diff
@@ -109,16 +109,37 @@ typedef struct _ARM64_VFP_STATE
 
 typedef struct _ARM64_UNWIND_PARAMS
 {
-    ULONG_PTR       ControlPc;
-    PULONG_PTR      LowLimit;
-    PULONG_PTR      HighLimit;
     PKNONVOLATILE_CONTEXT_POINTERS ContextPointers;
 } ARM64_UNWIND_PARAMS, *PARM64_UNWIND_PARAMS;
 
 #define UNWIND_PARAMS_SET_TRAP_FRAME(Params, Address, Size)
 
-#define UPDATE_CONTEXT_POINTERS(Params, RegisterNumber, Address)
-#define UPDATE_FP_CONTEXT_POINTERS(Params, RegisterNumber, Address)
+#define UPDATE_CONTEXT_POINTERS(Params, RegisterNumber, Address)                    \
```

Comment:
@jashook 

@jashook @janvorli 
There are only a few files touched which I expect to be shared by Windows.  Based on the build failures, my guess is it is this change to src/unwinder/arm64/unwinder_arm64.cpp

These lines probably need to be marked with #ifdef FEATURE_PAL

I was surprised to see these failures, but looks like there was no premerge run against Windows arm64.  It was run against arm and x86 only.

I'll create a PR if you want and you can see if it fixes WinArm64